Makefile

```diff
@@ -379,6 +386,26 @@ dist/calicoctl: $(CALICOCTL_FILES) vendor
 	    make binary && \
 	    chown -R $(shell id -u):$(shell id -u) dist'
 
+dist/calicoctl-darwin-amd64: $(BUILD_CALICOCTL_CONTAINER_MARKER) $(CALICOCTL_FILES) vendor
```

Comment:
Wouldn't it be better here to have these three communized as a single make target and to pass through the OS and ARCH envs into the container?